pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -1559,7 +1559,7 @@ func configToStatus(ctx *domainContext, config types.DomainConfig,
 			ds.Devtype = "hdd"
 		}
 		// map from i=1 to xvdb, 2 to xvdc etc
-		ds.Vdev = "xvd" + string(int('a')+i)
+		ds.Vdev = fmt.Sprintf("xvd%d", int('a')+i)
```

Comment:
@eriknordmark and @rvs (and anyone else who touched this), can you check that this is correct? The output should be `xvd97` then `xvd98`, then `xvd99`, etc.?

Reply:
No, the output should be xvda, xvdb, xvdc. Hence the type conversion from into to string.

Reply:
Ah, now I see it. OK, I will fix this.

Reply:
Fixed. Take a look please.